packages/arb-token-bridge-ui/src/components/common/HeaderNetworkInformation.tsx

```diff
@@ -14,7 +14,7 @@ export function HeaderNetworkInformation() {
     return null
   }
 
-  const networkName = getNetworkName(chain.id)
+  const networkName = getNetworkName(chain.id, true)
```

Comment:
It's better to do this kind of truncating via CSS: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Then when you select the text, and copy and paste you still get the full value.

Reply:
Comment applies for all similar cases in this pull request.